modules/sandboxed-containers-installing-operator-cli.adoc

```diff
@@ -0,0 +1,7 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-installing-operator-cli_{context}"]
+
+= Installing the Sandboxed Containers Operator using the CLI
```

Comment:
You can use your `Sandboxed Containers Operator` attribute here

Reply:
PV2/FCC Compliance - not to use attributes in the title